pinot-broker/src/main/java/org/apache/pinot/broker/routing/RoutingManager.java

```diff
@@ -616,4 +620,14 @@ void refreshSegment(String segment) {
       }
     }
   }
+
+  @Override
+  public Map<String, ServerInstance> getEnabledServerInstanceMap() {
+    return new HashMap<>(_enabledServerInstanceMap);
+  }
+
+  @Override
+  public RouteTable getRoutingTable(String tableName) {
+    return getRoutingTable(CALCITE_SQL_COMPILER.compileToBrokerRequest(String.format("SELECT * FROM %s", tableName)));
```

Comment:
This seem like temporary for debug ?

Reply:
this was to make sure the modification dependency on current pinot modules are at minimum. if accepted I think we can first merge this part of the change back to master branch and we don't have to do this hack

Reply:
This has been addressed in separate PR